docs/contributor/07-10-informer-cache.md

```diff
@@ -0,0 +1,3 @@
+# Informer's cache
+
+The controller manager uses informers with a cache. All observed resources (btpoperator, secret, configmap etc.) are stored in the cache. Because of out of memory risk, the cache is configured with a label selector `app.kubernetes.io/managed-by in (btp-manager,kcp-kyma-environment-broker)`. See [cache.go](/controllers/cache.go).
```

Comment:
The phrase "the out of memory risk" should be changed to:
- the out-of-memory risk if 'out-of-memory' just describes the type of risk
- the "out of memory" risk if 'out of memory' is the name used for the risk